VOICE-ASSISTANT.md

```diff
@@ -0,0 +1,201 @@
+# Use ha-sip to talk to your Home Assistant AI
+
+This guide shows you how to set up ha-sip to enable voice conversations with your Home Assistant AI assistant through phone calls.
+
+## Overview
+
+By combining ha-sip's phone capabilities with Home Assistant's AI conversation features, you can:
+- Call your Home and talk directly to your Home Assistant AI
+- Be called by Home Assistant if triggered by an automation
+- Ask questions and get spoken responses
+- Control your smart home devices using natural voice commands
+
+## Prerequisites
+
+### 1. ha-sip add-on installed and configured with at least one working SIP account
+
+ see [README.md](README.md). For this example the SIP account we are using is configured in "Listen mode".
+
+### 2. Home Assistant voice assistant pipeline with speech to text, conversation agent and text to speech.
+
+[![Open your Home Assistant instance and show your voice assistants.](https://my.home-assistant.io/badges/voice_assistants.svg)](https://my.home-assistant.io/redirect/voice_assistants/)
+
+The voice pipeline should be fast. If your hardware is a limitation, then you could use google free tiers for STT, LLM and TTS.
+
+### 3. Home Assistant Voice over IP integration enabled.
+
+[![Open your Home Assistant instance and start setting up a new integration.](https://my.home-assistant.io/badges/config_flow_start.svg)](https://my.home-assistant.io/redirect/config_flow_start/?domain=voip)
+
+In the configuration of the integration name a SIP username and a SIP port which is different from ha-sip, for example 5061.
```

Comment:
5061 is the standard TLS port for SIP, should change that to 5070 or so

Reply:
changed to 5070